Clarify client needs to reach the image proxy + code formatting update (#2939)
mattermost/docs: source/administration/image-proxy.rst

Proxy servers also provide a layer of caching, and can be made faster and more reliable than third-party sites. This caching 
also helps preserve posts by protecting them from dead images.

> When enabled, the image proxy needs to be publicly accessible to the Mattermost client, which requests the image directly from the image proxy.

Reviewer: From a followup to that conversation, we should probably mention that it needs to be accessible by both clients and the server.

```suggestion
When enabled, the image proxy needs to be publicly accessible to both the Mattermost client and server.
```

I'm not sure how to make the second part of the sentence work if we mention both of those though

.. image:: ../images/image-proxy.png

> The URL will be replaced with something similar to the following (ee `https://github.com/atmos/camo <https://github.com/atmos/camo>`__ for details):

Reviewer 2: ```suggestion
The URL will be replaced with something similar to the following (see `https://github.com/atmos/camo <https://github.com/atmos/camo>`__ for details):
```

Author: @amyblais Thanks! Updated
